src/bindgen/ir/cfg.rs

```diff
@@ -388,6 +388,10 @@ impl CfgWrite for Option<Cfg> {
                 cfg.write_condition(config, out);
                 out.new_line();
             }
+
+            for cfg in unmatched {
+                warn!("Missing `[defines]` entry for `{}` in cbindgen config.", cfg);
```

Comment:
I guess it's used here now :)

Reply:
It's slightly weird that this warns about missing defines where we generate code, but not when we don't (in the `!has_defines()` case)... Should `write_condition` be adapted to just skip unknown conditions (and simplify out unknown ones ahead of time, or replace them by `1` maybe) instead of the `matched_defines` business?

looks like it's easy to overlook that you need to call `matched_defines` to get a useful config. Anyway not a big deal :)